Userland/Applications/SoundPlayer/SoundPlayerWidgetAdvancedView.cpp

```diff
@@ -53,7 +54,18 @@ SoundPlayerWidgetAdvancedView::SoundPlayerWidgetAdvancedView(GUI::Window& window
     m_volume_icon = Gfx::Bitmap::load_from_file("/res/icons/16x16/audio-volume-medium.png"sv).release_value_but_fixme_should_propagate_errors();
     m_muted_icon = Gfx::Bitmap::load_from_file("/res/icons/16x16/audio-volume-muted.png"sv).release_value_but_fixme_should_propagate_errors();
 
-    m_visualization = m_player_view->add<BarsVisualizationWidget>();
+    auto visualization_widget = Config::read_string("SoundPlayer"sv, "Preferences"sv, "VisualizationWidget"sv, "bars"sv);
+    if (visualization_widget == "samples")
+        m_visualization = m_player_view->add<SampleWidget>();
+    if (visualization_widget == "album_cover")
+        m_visualization = m_player_view->add<AlbumCoverVisualizationWidget>([this]() {
+            return this->get_image_from_music_file();
+        });
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#omission-of-curly-braces-from-statement-blocks